package/src/components/Message/MessageSimple/MessageContent.tsx

```diff
@@ -377,6 +380,11 @@ const MessageContentWithContext = <
                 );
               case 'gallery':
                 return <Gallery key={`gallery_${messageContentOrderIndex}`} />;
+              case 'poll': {
+                const pollId = message.poll_id;
+                const poll = pollId && client.polls.fromState(pollId);
+                return pollId && poll ? <Poll key={`poll_${message.poll_id}`} poll={poll} /> : null;
```

Comment:
This seems like it is not used from the UI prop that you introduced, but the component is directly imported.

Reply:
Not sure what you mean - but if you're referring to the component itself, that's not the one that users can override. This is a HoC that provides the polls with everything needed to function properly.